shared/wallets/wallet/settings/container.js

```diff
@@ -17,7 +17,7 @@ import * as WalletsGen from '../../../actions/wallets-gen'
 const mapStateToProps = (state: TypedState, {routeProps}) => {
   const accountID = routeProps.get('accountID')
   const account = Constants.getAccount(state, accountID)
-  const name = account.name || accountID || account.accountID
+  const name = account.name || Constants.getAccountName(account) || accountID || account.accountID
```

Comment:
i think we decided this should always be account.name and we're going to make it such that new wallets always have to have one

Reply:
ok